.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2.1
+
+orbs:
+  aws-eks: circleci/aws-eks@1
+  gcp-cli: circleci/gcp-cli@1
+  go: circleci/go@1
+
+jobs:
+  checkout:
+    executor:
+      name: go/default
+      tag: '1.17'
+    steps:
+      - checkout
+      - go/mod-download-cached
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - go
+            - project
+
+  build:
+    executor:
+      name: go/default
+      tag: '1.17'
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Building package
+          command: make build
+
+  test:
+    executor:
+      name: go/default
+      tag: '1.17'
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Run fmt-test
+          command: make fmt-test
+      - run:
+          name: Installing golangci-lint
+          command: make install-golangci-lint
+      - run:
+          name: Run lint
+          command: make lint
```

Comment:
Looking at the github actions, i guess `lint` can be removed

Reply:
I think so too. It would be redundant, and static analysis is a really good fit for GitHub Actions since you get the result directly in the code diff.

I'd remove that part. (Unless some CircleCI integration can provide the same output)

Reply:
removed!